packages/main/src/index.ts

```diff
@@ -136,6 +136,9 @@ import { SideNavigationItem } from './dist/SideNavigationItem';
 import { SideNavigationSubItem } from './dist/SideNavigationSubItem';
 import { Size } from './dist/Size';
 import { Slider } from './dist/Slider';
+import { Splitter } from './dist/Splitter';
```

Comment:
Is Splitter a component the user should be able to consume? If so please add jsDoc comments there as well, otherwise use it as internal component only and therefore don't export it to our library.